R/collections.R

```diff
@@ -672,8 +672,8 @@ article_footer_html <- function(site_dir, site_config, collection, article) {
     share <- tags$span(class = "article-sharing",
       HTML("Share: &nbsp;"),
       tagList(lapply(share_services, function(service) {
-        tags$a(href = sharing_link(service),
-          tag("i", list(class = sprintf("fab fa-%s", service)))
+        tags$a(href = sharing_link(service), `aria-label` = service,
```

Comment:
And what `aria-label` is supposed to mean ? 
Should we say `"twitter icon"` instead of just twitter ? Or the aim of the link maybe like `"Share on twitter` ? 

it would be easy to tweak so asking as I don't know the meaning of those `aria-*` attributes.

```suggestion
        tags$a(href = sharing_link(service), `aria-label` = sprintf("share on %s", service),
```

Reply:
I thought about that too—I like your suggestion. At first I thought it would be redundant, since it says "Sharing" in text beforehand, but I like this way.